### PR TITLE
Create Footer Form

### PR DESCRIPTION
**Pull request by rebaz36**
SetUp Formspree.io
Create footer form
stylize form

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -189,8 +189,31 @@ <h3>skills</h3>
...
+        </div>
+
+        <input type="text" name="name" required maxlength="30" placeholder="Name...">
+
+        <input type="email" name="email" required placeholder="Email...">
+
+        <textarea class="form-text" name="message" maxlength="500" required placeholder="write your message here..."></textarea>
+
+        <!-- your other form fields go here -->
+        <button class="form-button" type="submit">Get in Touch</button>
```
Comment (mugishasam123) [inline index.html:209]: - [x] kindly add button interactions to this button like hover, focus, etc .which will enhance the user's interaction.

**Review by mugishasam123 — CHANGES_REQUESTED** [review]
Hi @rebaz36  👋 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

ALT
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights

✔️ No linter errors
✔️ Github flow was used correctly
✔️ Descriptive PR title 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (rebaz36) [reply index.html:209]: @mugishasam123  thanks for the suggestion

**Review by victorokoroji — APPROVED** [review]
Hi @rebaz36 , 

# 🟢  Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**